modules/reference/pages/rpk/rpk-cluster/rpk-cluster-storage-mount.adoc

```diff
@@ -0,0 +1,59 @@
+= rpk cluster storage mount
+
+Mount a topic to the Redpanda cluster from glossterm:tiered-storage[].
+
+This command mounts a topic in the Redpanda cluster using log segments stored in Tiered Storage. 
+
+The topic may be optionally renamed with the `--to` flag.
+
+Requirements:
+
+- Tiered Storage must be enabled.
```

Comment:
link to https://docs.redpanda.com/current/manage/tiered-storage/#enable-tiered-storage